webrtc.html

```diff
@@ -4461,6 +4461,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     is designed to minimize occurrence of images with with letter
     box or or pillow boxing. The media MUST NOT be upscaled to
     create fake data that did not occur in the input source.</p>
+    <p>When video is rescaled, for example for certain combinations
+    of width or height and
+    <code><a data-link-for="RTCRtpEncodingParameters">
+    scaleResolutionDownBy</a></code> values, situations when the result must
+    be rounded to obtain an integer result may occur. In such situations the
+    user agent MUST round up to the nearest integer value.</p>
```

Comment:
Decision at the virtual interim was to change from "ceil" to "floor". The reasoning is that it's easier for scaling algorithms to handle weird resolutions by cropping off lines of pixels at the edge as needed. For example, it's easy to scale a 101x101 image down to 50x50 by removing one row/column of pixels, then doing 2:1 scaling (where every output pixel is determined by 4 input pixels).